project-codeflare/codeflare-sdk
# Updated codeflare-torchx version

# Issue link


[RHOAIENG-3327](https://issues.redhat.com/browse/RHOAIENG-3327)
# What changes have been made

Added updated python package [codeflare-torchx](https://pypi.org/project/codeflare-torchx/0.6.0.dev2/)
# Verification steps



* run `poetry build` and `pip install --force-reinstall /path/to/your/dist/codeflare_sdk-0.0.0.dev0-py3-none-any.whl`
* run `pip freeze | grep torchx`
* Verify codeflare-torchx version is correct

Follow the steps outlined in this PR 
## Checks
- Testing Strategy
